franklin/github/views.py

```diff
@@ -136,42 +65,52 @@ def repository_list(request):
             github_repos = request.user.details.get_user_repos()
             # TODO - return error from github
             request.user.details.update_repos_for_user(github_repos)
-        sites = request.user.details.sites.all()
+        sites = request.user.details.sites.filter(is_active=True).all()
         serializer = SiteSerializer(sites, many=True)
         return Response(serializer.data)
     elif request.method == 'POST':
+        # TODO - #53 will have us refactoring all errors returned from our
+        # external dependencies. It will need to handle cases like this where
+        # we get a potential non-error like this one.
```

Comment:
Some of the below logic is a little ugly. Will address most of it when we do that refactor to throw errors low and catch them high.